src/main/java/svc/controllers/SMSController.java

```diff
@@ -0,0 +1,45 @@
+package svc.controllers;
+
+import java.io.IOException;
+import com.twilio.twiml.MessagingResponse;
+import com.twilio.twiml.TwiMLException;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import svc.managers.*;
+import svc.models.*;
+
+@RestController
+@EnableAutoConfiguration
+@RequestMapping("/sms")
+public class SMSController {
+	@Inject
+	SMSManager smsManager;	
+		
+	@ResponseBody
+	@RequestMapping(method = RequestMethod.POST)
+	void GetMessage(@ModelAttribute TwimlMessageRequest twimlMessageRequest, HttpServletResponse response, HttpSession session) throws IOException{
+		session.setMaxInactiveInterval(30*60); //set session timeout to 30 minutes
+		
+		MessagingResponse twimlResponse = smsManager.getTwimlResponse(twimlMessageRequest, session);
+
+	    response.setContentType("application/xml");
```

Comment:
What seems wrong about this?  Is it the empty lines between the pieces of code.  If so, it is because I deleted a bunch of code and moved it into the manager.  I left it spaced like this for easier reading.  But I can change that if this is what you are referring to.